Iceberg-TipUI/IceTipLocalRepositoryPanel.class.st

```diff
@@ -86,11 +86,9 @@ IceTipLocalRepositoryPanel >> validate [
 
 	self
 		assert: self location isNotNil
-		description: 'Project location must exist!'.
+		description: 'Please provide a project location!'.
 	self
 		assert: self location exists
 		description: 'Project location must exist!'.
-	self
-		assert: (IceRepositoryCreator isGitRoot: self location)
```

Comment:
Is it normal that this assertion is removed?

Reply:
sorry it was link to the fix of another issue mb